app/main/views/marketplace.py

```diff
@@ -103,8 +121,18 @@ def get_brief_by_id(framework_framework, brief_id):
     return render_template(
         'brief.html',
         brief=brief,
-        started_brief_responses_count=started_brief_responses_count,
-        completed_brief_responses_count=completed_brief_responses_count,
+        started_brief_responses_stats={
+            'started_sme_brief_responses_count': started_sme_brief_responses_count,
+            'started_large_brief_responses_count': started_large_brief_responses_count,
+            'started_brief_responses_total': started_sme_brief_responses_count + started_large_brief_responses_count
```

Comment:
I think it would be reasonable to just name the keys `sme`, `large`, and `total`. It's apparent that they're started brief response counts because of the name of the dict they're stored in, so repeating all of that detail feels a bit redundant and makes it harder to parse.